### ISSUE TITLE
Add support for the Mica material

### ISSUE THREAD

**Issue by DHowett** [microsoft/terminal#10509]
Depends on #10508

**Comment by sylveon** [comment]
Blocked by https://github.com/microsoft/microsoft-ui-xaml/issues/5319

**Comment by DHowett** [comment]
Oh no. Thanks.

**Comment by Extrimis** [comment]
https://github.com/microsoft/microsoft-ui-xaml/issues/5319#issuecomment-894008314

**Comment by dongle-the-gadget** [comment]
Or you can reference WinUI's source code and create a composition brush for Mica.

**Comment by Extrimis** [comment]
Have you tried using Compositor.TryCreateBlurredWallpaperBackdropBrush() , it gives the blurred wallpaper brush that is used for creating mica. It is also supported in windows.ui.composition

**Comment by lazerl0rd** [comment]
There's also the "Tabbed" material used by Explorer [on the last few Insider builds], which would be nice to have for the tab bar's background:
![image](https://user-images.githubusercontent.com/24303423/161751714-a7ca4d10-b769-46ab-bbf4-2c011493e0d6.png)

**Comment by dongle-the-gadget** [comment]
Not sure about its implementation.

**Comment by zadjii-msft** [comment]
Since I'm tracking this here: `dev/migrie/f/no-custom-caption-btns` has the weird version of the terminal with no MinMaxClose control, only the DWM frame. I think that's a dead end though.

**Comment by plashenkov** [comment]
Hey! Thanks for adding Mica!

It's worth noting that you have to move opacity slider to 0 to have Mica effect on a client terminal area.

But you'll see a horizontal line between the client area and the title bar in that case, and the active tab and client area aren't seamless.

To solve this, we can make the title bar darker, and, since we can't have completely transparent elements (tab) over semitransparent (darker title bar), we can add some white to the active tab to workaround this.

```jsonc
{
  "profiles": {
    "defaults": {
      // ...attributes...
      "opacity": 0
    }
  },
  "theme": "Mica",
  "themes": [
    {
      "name": "Mica",
      "window": {
        "applicationTheme": "dark",
        "useMica": true
      },
      "tabRow": {
        "background": "#00000043", // darker
        "unfocusedBackground": "#00000043" // darker
      },
      "tab": {
        "background": "#FFFFFF09", // lighter
        "unfocusedBackground": "#00000000" // transparent
      }
    }
  ]
}
```

Finally, this is how it looks now:

![terminal](https://github.com/microsoft/terminal/assets/2331932/8e5ae981-7ff6-47eb-8739-c10a3f913fbb)

**Comment by plashenkov** [comment]
Hey @Void48 

I've already posted everything needed. This should really be enough, no need to expose other parts of my config. If you've already added these settings, what do you get?

**Comment by DunderMiko** [comment]
Hi, I can't quite get the seem to disappear no matter what value I set. The closest i got was at:

  "theme": "Mica",
  "themes": [
    {
      "name": "Mica",
      "window": {
        "applicationTheme": "dark",
        "useMica": true
      },
      "tabRow": {
        "background": "#00000043", // darker
        "unfocusedBackground": "#00000043" // darker
      },
      "tab": {
        "background": "#FFFFFF06", // lighter
        "unfocusedBackground": "#00000000" // transparent
      }
    }
  ]
}

Any tips?

<img width="363" alt="Image" src="https://github.com/user-attachments/assets/af2f2acc-69ce-4214-b528-a40b62ea7417" />

**Comment by edge261** [comment]
> Actually I figured it out [@plashenkov](https://github.com/plashenkov) 👍🏻

how you did that? I still get that difference :( 

<img width="948" height="153" alt="Image" src="https://github.com/user-attachments/assets/6d820bd4-1e05-42fb-9753-d4d834493d2a" />

**Comment by plashenkov** [comment]
@edge261 On some backgrounds I can see the difference too. I think we cannot eliminate it completely, unfortunately.

**Comment by DunderMiko** [comment]
> [@edge261](https://github.com/edge261) On some backgrounds I can see the difference too. I think we cannot eliminate it completely, unfortunately.


A workaround I used for this is to set a custom color for the tab, I used color picker and set teh color to the nearest one fone in the mica window beneath the tab.

<img width="661" height="212" alt="Image" src="https://github.com/user-attachments/assets/7aa731c1-a1b7-4d67-bd08-5329a644b5b1" />
